go.mod

```diff
@@ -1,24 +1,27 @@
 module github.com/tillitis/tkey-sign-cli
 
-go 1.19
+go 1.21
+
+toolchain go1.22.2
```

Comment:
Should we skip the toolchain part?

Reply:
Oops. Yes, I think so, at least for now, so CI can build. Removed.